ui/src/channels/NewChannel/NewChannelDetails.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import cn from 'classnames';
+import { capitalize } from 'lodash';
+import { useFormContext } from 'react-hook-form';
+import ChannelPermsSelector from '@/groups/GroupAdmin/AdminChannels/ChannelPermsSelector';
+import { ChannelType, ChannelVisibility } from '@/types/groups';
+import { useCalm } from '@/state/settings';
+import LoadingSpinner from '@/components/LoadingSpinner/LoadingSpinner';
+import Lock16Icon from '@/components/icons/Lock16Icon';
+import LockOpen16Icon from '@/components/icons/LockOpen16Icon';
+
+interface NewChannelDetailsProps {
+  edit?: boolean;
+  channelType: ChannelType;
+  visibility: ChannelVisibility;
+  goToNextStep: () => void;
+  goToPrevStep: () => void;
+}
+
+export default function NewChannelDetails({
+  edit,
+  goToNextStep,
+  goToPrevStep,
+  visibility,
+  channelType,
+}: NewChannelDetailsProps) {
+  const { register, formState, watch } = useFormContext();
+  const calm = useCalm();
+  const watchTitle = watch('meta.title');
+  const channelTypeName =
+    channelType === 'chat'
+      ? 'Chat'
+      : channelType === 'heap'
+      ? 'Gallery'
+      : 'Notebook';
+
+  return (
+    <div>
+      <div className="sm:w-96">
+        <header className="mb-6 flex items-center">
+          <h2 className="text-lg font-bold">Customize: {watchTitle}</h2>
+        </header>
+      </div>
+      <div className="flex flex-col space-y-6">
+        <label className="font-semibold">
+          {channelTypeName} Description
+          <textarea
+            // TODO: set sane maxLength
+            {...register('meta.description', { maxLength: 300 })}
```

Comment:
although this isn't marked as required, the form seems to require me to put something here. I don't think this is supposed to be required is it?